tests/baselines/reference/findAllRefsForStringLiteral2.baseline.jsonc

```diff
@@ -0,0 +1,47 @@
+// === /a.ts ===
+// /**
+//  * @type {"[|foo|]" | "bar"}
```

Comment:
I think find-all-refs should ignore `@type` in ts files and not offer completions either.